apps/skilavottord/ws/src/app/modules/vehicle/vehicleAppSys.resolver.ts

```diff
@@ -35,7 +35,13 @@ export class VehicleAppSysResolver {
       input.permno,
     )
     if (!vehicle) {
-      throw new NotFoundException(`User does not have this vehicle`)
+      logger.error(
+        'User ' +
+          user.nationalId +
+          ' does not own this vehicle ' +
+          input.permno,
+      )
+      throw new NotFoundException(`User does not own this vehicle`)
```

Comment:
Please log nicely with template strings, any useful details in object.
```suggestion
logger.error(`User ${user.nationalId} does not own the requested vehicle`, { permno: input.permno, user })
return null
```

Reply:
Fixed according to suggestion